File: libs/common/models/vw-location-website.ts



export class VwLocationWebsite {
    public locationWebsiteId: number = 0;
    public websiteUsage: string = '';
    public listSequence: number;
    public websiteUrl: string = '';
    public locationId: number;
    public businessStartDate: Date;
    public businessEndDate: Date;

    constructor(init? : Partial<VwLocationWebsite>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/EntityFramework/VwLocationWebsite.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class VwLocationWebsite
    {
        public int LocationWebsiteId { get; set; }
        public string WebsiteUsage { get; set; }
        public int? ListSequence { get; set; }
        public string WebsiteUrl { get; set; }
        public int? LocationId { get; set; }
        public DateTime? BusinessStartDate { get; set; }
        public DateTime? BusinessEndDate { get; set; }
    }
}
